canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.jsx

```diff
@@ -265,8 +266,26 @@ class DropDown extends React.Component {
 					disabled={this.props.state === STATES.DISABLED || this.disableEmptyListDropdown}
 					type="default"
 					items={dropDown.options}
+					itemToElement={(item) => (item ? (
+						<span className="icon-support" style={{ color: "black" }}>
+							<span>{ item.value }</span>
+							<Password style={{ float: "right" }} />
+						</span>
+					) : (
+						" "
+					))
+					}
 					onChange={this.handleChange}
 					selectedItem={dropDown.selectedOption}
+					itemToString={(item) => (item ? (
+						<span className="icon-support" style={{ color: "black", display: "flex", justifyContent: "space-between" }}>
```

Comment:
We dont specify inline styles unless its absolutely necessary. We set styles in scss files using class names that are unique to this class so that there are no chances of any classname overlaps with those of the application